ext/node/path/constants.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright the Browserify authors. MIT License.
+// Ported from https://github.com/browserify/path-browserify/
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+#![allow(warnings)]
```

Comment:
Unsure if this is better than adding `#![allow(unused)]` to every constant which would double the file size.. however this mutes helpful clippy warnings.. thoughts ?